gui/src-tauri/src/main.rs

```diff
@@ -229,9 +229,15 @@ fn main() {
 			}
 			Ok(())
 		})
-		//
+		.on_window_event(|e| match e.event() {
```

Comment:
Can you ``#[cfg(windows)]`` this, or test it in other OSes?

Reply:
you can if you wish. But this is a chromium problem presumably, so I think its safer to keep it unconditional. Who knows what the OS is using as their webview.

Reply:
I have had 0 issues in Linux in my experience btw

Reply:
> Who knows what the OS is using as their webview.

both linux and macos currently use a webkit based webview. It shouldn't have any (positive) effect on these platforms. Buuut maybe it helps on Linux where some users see white flickering when resizing (i don't have that problem on my system so i can't test it)
tldr: `#[cfg(windows)]` should be fine, or even recommended.